# German translation of update-manager.
# Copyright (C) 2005 Michiel Sikkes  <michiel@eyesopened.nl>
# This file is distributed under the same license as the update-manager package.
# Initial version by an unknown artist.
# Frank Arnold <frank@scirocco-5v-turbo.de>, 2005.
#
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2013-05-22 21:30+0100\n"
"PO-Revision-Date: 2013-06-04 08:35+0000\n"
"Last-Translator: Hendrik Knackstedt <Unknown>\n"
"Language-Team: German GNOME Translations <gnome-de@gnome.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2014-04-23 06:51+0000\n"
"X-Generator: Launchpad (build 16985)\n"
"Language: \n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:211 ../DistUpgrade/distro.py:441
#, python-format
msgid "Server for %s"
msgstr "Server für %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:229 ../DistUpgrade/distro.py:235
#: ../DistUpgrade/distro.py:251
msgid "Main server"
msgstr "Hauptserver"

#: ../DistUpgrade/distro.py:255
msgid "Custom servers"
msgstr "Benutzerdefinierte Server"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Eintrag für sources.list konnte nicht erstellt werden"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Es konnten keine Paketdateien gefunden werden, möglicherweise ist dies keine "
"Ubuntu-CD oder die falsche Architektur?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Hinzufügen der CD gescheitert"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Beim Hinzufügen der CD trat ein Fehler auf. Die Systemaktualisierung wird "
"abgebrochen. Bitte melden Sie dies als einen Fehler, wenn Sie eine "
"offizielle Ubuntu-CD verwendet haben.\n"
"\n"
"Die Fehlermeldung war:\n"
"%s"

#: ../DistUpgrade/DistUpgradeCache.py:152
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Defektes Paket entfernen"
msgstr[1] "Defekte Pakete entfernen"

#: ../DistUpgrade/DistUpgradeCache.py:155
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Das Paket »%s« befindet sich in einem inkonsistenten Zustand und muss neu "
"installiert werden, allerdings kann dafür kein Archiv gefunden werden. "
"Möchten Sie das Paket nun entfernen, um fortzufahren?"
msgstr[1] ""
"Die Pakete »%s« befinden sich in einem inkonsistenten Zustand und müssen neu "
"installiert werden, allerdings können dafür keine Archive gefunden werden. "
"Wollen Sie diese Pakete nun entfernen,  um fortzufahren?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:256
msgid "The server may be overloaded"
msgstr "Der Server ist möglicherweise überlastet"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid "Broken packages"
msgstr "Defekte Pakete"

#: ../DistUpgrade/DistUpgradeCache.py:370
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Ihr System enhält defekte Pakete, die mit dieser Anwendung nicht repariert "
"werden können. Bitte verwenden Sie »Synaptic« oder »apt-get« zur Reparatur "
"der Pakete, bevor Sie fortfahren."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Ein unlösbares Problem ist während der Systemaktualisierung aufgetreten:\n"
"%s\n"
"\n"
" Dies kann folgende Ursachen haben:\n"
" * Sie möchten auf eine Vorabveröffentlichung von Ubuntu aktualisieren\n"
" * Sie benutzen eine Vorabveröffentlichung von Ubuntu\n"
" * Sie benutzen Softwarepakete, die nicht aus einer offiziellen Ubuntu-"
"Quelle stammen\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:648
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Dies ist wahrscheinlich ein vorübergehendes Problem.\n"
"Bitte versuchen Sie es zu einem späteren Zeitpunkt erneut."

#: ../DistUpgrade/DistUpgradeCache.py:651
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Falls nichts von dem zutrifft, melden Sie diesen Fehler bitte, indem Sie den "
"Befehl »ubuntu-bug ubuntu-release-upgrader-core« in einem Terminal ausführen."

#: ../DistUpgrade/DistUpgradeCache.py:656
msgid "Could not calculate the upgrade"
msgstr ""
"Es konnte nicht ermittelt werden, welche Systemaktualisierungen verfügbar "
"sind"

#: ../DistUpgrade/DistUpgradeCache.py:707
msgid "Error authenticating some packages"
msgstr "Fehler bei der Echtheitsprüfung einiger Pakete"

#: ../DistUpgrade/DistUpgradeCache.py:708
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Die Echtheit einiger Pakete konnte nicht bestätigt werden. Dies kann an "
"vorübergehenden Netzwerkproblemen liegen. Bitte versuchen Sie es zu einem "
"späteren Zeitpunkt noch einmal. Unten stehend sehen Sie eine Liste "
"derjenigen Pakete, deren Echtheit nicht bestätigt werden konnte."

#: ../DistUpgrade/DistUpgradeCache.py:728
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Das Paket »%s« ist zum Löschen vorgesehen, wurde aber durch das System "
"gesperrt."

#: ../DistUpgrade/DistUpgradeCache.py:732
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Das unbedingt notwendige Paket »%s« ist zum Löschen vorgesehen."

#: ../DistUpgrade/DistUpgradeCache.py:741
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Es wird versucht, die auf der schwarzen Liste stehende Version »%s« zu "
"installieren"

#: ../DistUpgrade/DistUpgradeCache.py:859
#, python-format
msgid "Can't install '%s'"
msgstr "»%s« kann nicht installiert werden"

#: ../DistUpgrade/DistUpgradeCache.py:860
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Es war nicht möglich, ein erforderliches Paket zu installieren. Bitte melden "
"Sie diesen Fehler, indem Sie im Terminal den Befehl »ubuntu-bug ubuntu-"
"release-upgrader-core« eingeben."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:871
msgid "Can't guess meta-package"
msgstr "Metapaket konnte nicht bestimmt werden"

#: ../DistUpgrade/DistUpgradeCache.py:872
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Ihr System enthält weder ein »ubuntu-desktop«-, ein »kubuntu-desktop«-, noch "
"ein »edubuntu-desktop«-Paket und es war nicht möglich zu ermitteln, welche "
"Ubuntu-Version Sie verwenden.\n"
" Bitte installieren Sie eines der oben genannten Pakete über Synaptic oder "
"apt-get bevor Sie fortfahren."

#: ../DistUpgrade/DistUpgradeController.py:126
msgid "Reading cache"
msgstr "Zwischenspeicher wird gelesen"

#: ../DistUpgrade/DistUpgradeController.py:230
msgid "Unable to get exclusive lock"
msgstr "Kein exklusiver Zugriff möglich"

#: ../DistUpgrade/DistUpgradeController.py:231
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Dies bedeutet üblicherweise, dass die Aktualisierungsverwaltung oder eine "
"andere Paketverwaltung, wie z.B. »Synaptic«, »apt-get« oder »aptitude«, "
"bereits läuft. Bitte beenden Sie zuerst die laufende Anwendung."

#: ../DistUpgrade/DistUpgradeController.py:289
msgid "Upgrading over remote connection not supported"
msgstr ""
"Die Systemaktualisierung über eine entfernte Verbindung wird nicht "
"unterstützt"

#: ../DistUpgrade/DistUpgradeController.py:290
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Sie versuchen die Systemaktualisierung über eine entfernte ssh-Verbindung "
"mit einer Oberfläche durchzuführen, die dies nicht unterstützt. Bitte "
"versuchen Sie die textbasierte Systemaktualisierung mittels »do-release-"
"upgrade«.\n"
"\n"
"Die Systemaktualisierung wird nun beendet. Bitte versuchen Sie es ohne ssh."

#: ../DistUpgrade/DistUpgradeController.py:304
msgid "Continue running under SSH?"
msgstr "Soll die Sitzung über SSH fortgesetzt werden?"

#: ../DistUpgrade/DistUpgradeController.py:305
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Diese Sitzung läuft offenbar über SSH. Es wird davon abgeraten, eine "
"Systemaktualisierung über SSH durchzuführen, da im Fehlerfall eine "
"Wiederherstellung schwierig sein kann.\n"
"\n"
"Wenn Sie fortfahren, wird ein zusätzlicher SSH-Dienst auf Port »%s« "
"gestartet.\n"
"Sind Sie sicher, dass Sie fortfahren möchten?"

#: ../DistUpgrade/DistUpgradeController.py:319
msgid "Starting additional sshd"
msgstr "Zusätzlicher SSH-Server wird gestartet"

#: ../DistUpgrade/DistUpgradeController.py:320
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Um eine Wiederherstellung im Fall einer Fehlfunktion zu vereinfachen, wird "
"ein zusätzlicher sshd-Prozess auf Port »%s« gestartet. Wenn es Probleme mit "
"dem aktuellen sshd-Prozess gibt, können Sie sich mit dem zusätzlichen "
"Prozess verbinden.\n"

#: ../DistUpgrade/DistUpgradeController.py:328
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Wenn Sie gerade eine Firewall ausführen, müssen Sie diesen Port vielleicht "
"manuell öffnen. Da es möglicherweise gefährlich ist, wird dies nicht "
"automatisch gemacht. Sie können den Port z.B. so öffnen:\n"
"»%s«"

#: ../DistUpgrade/DistUpgradeController.py:400
#: ../DistUpgrade/DistUpgradeController.py:445
msgid "Can not upgrade"
msgstr "Systemaktualisierung kann nicht durchgeführt werden"

#: ../DistUpgrade/DistUpgradeController.py:401
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"Eine Systemaktualisierung von »%s« auf »%s« wird von diesem Werkzeug nicht "
"unterstützt."

#: ../DistUpgrade/DistUpgradeController.py:410
msgid "Sandbox setup failed"
msgstr "Einrichten der Sandbox gescheitert"

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "It was not possible to create the sandbox environment."
msgstr "Die Sandbox-Umgebung konnte nicht erstellt werden."

#: ../DistUpgrade/DistUpgradeController.py:417
msgid "Sandbox mode"
msgstr "Sandbox-Modus"

#: ../DistUpgrade/DistUpgradeController.py:418
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Diese Systemaktualisierung wird in einer Testumgebung (im Sandbox-Modus) "
"ausgeführt. Alle Änderungen werden in »%s« gespeichert und beim nächsten "
"Neustart verloren sein.\n"
"*Alle* von jetzt an bis zum nächsten Neustart gemachten Änderungen in System-"
"Verzeichnissen – *sind nicht permanent*."

#: ../DistUpgrade/DistUpgradeController.py:446
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Ihre Python-Installation ist beschädigt. Bitte korrigieren Sie die "
"Verknüpfung »/usr/bin/python«."

#: ../DistUpgrade/DistUpgradeController.py:472
msgid "Package 'debsig-verify' is installed"
msgstr "Das Paket »debsig-verify« ist installiert"

#: ../DistUpgrade/DistUpgradeController.py:473
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Die Systemaktualisierung kann ohne die Entfernung dieses Paketes nicht "
"fortgesetzt werden. \n"
"Bitte entfernen Sie es zunächst mit Synaptic oder »apt-get remove debsig-"
"verify« und starten Sie danach die Systemaktualisierung neu."

#: ../DistUpgrade/DistUpgradeController.py:485
#, python-format
msgid "Can not write to '%s'"
msgstr "Auf »%s« kann nicht geschrieben werden."

#: ../DistUpgrade/DistUpgradeController.py:486
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Die Systemaktualisierung kann nicht fortgesetzt werden, da es nicht möglich "
"ist, in das System-Verzeichnis »%s« zu schreiben.\n"
"Stellen Sie bitte sicher, dass das System-Verzeichnis beschreibbar ist."

#: ../DistUpgrade/DistUpgradeController.py:497
msgid "Include latest updates from the Internet?"
msgstr "Die neuesten Aktualisierungen aus dem Internet mit einbeziehen?"

#: ../DistUpgrade/DistUpgradeController.py:498
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Das Aktualisierungssystem kann auf das Internet zugreifen, um die neuesten "
"Pakete automatisch herunterzuladen und diese während der "
"Systemaktualisierung zu installieren. Falls Sie eine Netzwerkverbindung "
"haben, ist das dringend empfohlen.\n"
"\n"
"Die Systemaktualisierung wird länger dauern, allerdings ist das System nach "
"der Fertigstellung auf dem neuesten Stand. Sie können sich dagegen "
"entscheiden, sollten aber die neuesten Aktualisierungen möglichst bald nach "
"der Systemaktualisierung installieren.\n"
"Falls Sie sich für »Nein« entscheiden, wird auf das Netzwerk nicht "
"zugegriffen."

#: ../DistUpgrade/DistUpgradeController.py:729
#, python-format
msgid "disabled on upgrade to %s"
msgstr "Bei Aktualisierung zu %s deaktiviert"

#: ../DistUpgrade/DistUpgradeController.py:762
msgid "No valid mirror found"
msgstr "Es wurde kein gültiger Spiegelserver gefunden"

#: ../DistUpgrade/DistUpgradeController.py:753
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Beim Überprüfen Ihrer Softwarequelleninformationen wurde kein "
"Spiegelservereintrag für die Systemaktualisierung gefunden. Dies kann "
"passieren, wenn Sie einen internen Spiegelserver verwenden oder wenn dessen "
"Informationen veraltet sind.\n"
"\n"
"Soll Ihre »sources.list« trotzdem neu geschrieben werden? Wenn Sie »Ja« "
"wählen, werden alle Einträge von »%s« auf »%s« aktualisiert.\n"
"Wählen Sie hingegen »Nein«, wird die Systemaktualisierung abgebrochen."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:783
msgid "Generate default sources?"
msgstr "Sollen die Standardeinträge für Paketquellen eingetragen werden?"

#: ../DistUpgrade/DistUpgradeController.py:784
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Nach dem Überprüfen Ihrer »sources.list« wurde kein gültiger Eintrag für "
"»%s« gefunden.\n"
"Sollen Standardeinträge für »%s« hinzugefügt werden? Wenn Sie »Nein« wählen, "
"wird die Systemaktualisierung abgebrochen."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Repository information invalid"
msgstr "Die Informationen über verfügbare Paketquellen sind ungültig"

#: ../DistUpgrade/DistUpgradeController.py:820
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Das Aktualisieren der Paketquellen-Informationen ließ eine ungültige Datei "
"entstehen, weshalb der Fehlermeldevorgang gestartet wird."

#: ../DistUpgrade/DistUpgradeController.py:827
msgid "Third party sources disabled"
msgstr "Paketquellen von Drittanbietern deaktiviert"

#: ../DistUpgrade/DistUpgradeController.py:828
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Einige Paketquellen von Drittanbietern wurden deaktiviert. Sie können diese "
"nach der Systemaktualisierung mit dem Programm »Software-Paketquellen« oder "
"mit »Synaptic« wieder aktivieren."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paket in inkonsistentem Zustand"
msgstr[1] "Pakete in inkonsistentem Zustand"

#: ../DistUpgrade/DistUpgradeController.py:873
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Das Paket »%s« ist in einem inkonsistentem Zustand und muss neu installiert "
"werden, aber es wurde kein Archiv dafür gefunden. Bitte installieren Sie das "
"Paket erneut manuell oder entfernen Sie es vom System."
msgstr[1] ""
"Die Pakete »%s« sind in einem inkonsistentem Zustand und müssen neu "
"installiert werden, aber es wurden keine Archive dafür gefunden. Bitte "
"installieren Sie die Pakete erneut manuell oder entfernen Sie sie vom System."

#: ../DistUpgrade/DistUpgradeController.py:921
msgid "Error during update"
msgstr "Fehler während der Aktualisierung"

#: ../DistUpgrade/DistUpgradeController.py:922
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Bei der Aktualisierung trat ein Problem auf. Dies ist häufig auf "
"Netzwerkprobleme zurückzuführen. Bitte überprüfen Sie Ihre "
"Netzwerkverbindung und versuchen Sie es erneut."

#: ../DistUpgrade/DistUpgradeController.py:931
msgid "Not enough free disk space"
msgstr "Nicht genug freier Festplattenspeicher verfügbar"

#: ../DistUpgrade/DistUpgradeController.py:932
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Die Aktualisierung wurde abgebrochen. Die Aktualisierung benötigt insgesamt "
"%s freien Speicherplatz auf Laufwerk »%s«. Bitte geben Sie noch mindestens "
"%s Speicherplatz auf dem Laufwerk »%s« frei. Leeren Sie den Müll und "
"entfernen Sie temporäre Pakete von früheren Installationen, indem sie »sudo "
"apt-get clean« ausführen."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:961
#: ../DistUpgrade/DistUpgradeController.py:1743
msgid "Calculating the changes"
msgstr "Änderungen werden berechnet"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:993
msgid "Do you want to start the upgrade?"
msgstr "Möchten Sie die Systemaktualisierung starten?"

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid "Upgrade canceled"
msgstr "Systemaktualisierung abgebrochen"

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Die Systemaktualisierung wird jetzt abgebrochen und der ursprüngliche "
"Zustand des Systems wiederhergestellt. Sie können die Systemaktualisierung "
"später fortsetzen."

#: ../DistUpgrade/DistUpgradeController.py:1073
#: ../DistUpgrade/DistUpgradeController.py:1197
msgid "Could not download the upgrades"
msgstr "Die Aktualisierungen konnten nicht heruntergeladen werden"

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Die Systemaktualisierung wurde abgebrochen. Bitte überprüfen Sie Ihre "
"Internetverbindung oder Ihr Installationsmedium und versuchen Sie es erneut. "
"Alle bis jetzt heruntergeladenen Dateien wurden gespeichert."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1148
#: ../DistUpgrade/DistUpgradeController.py:1185
#: ../DistUpgrade/DistUpgradeController.py:1290
msgid "Error during commit"
msgstr "Fehler beim Anwenden"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1150
#: ../DistUpgrade/DistUpgradeController.py:1187
#: ../DistUpgrade/DistUpgradeController.py:1329
msgid "Restoring original system state"
msgstr "Ursprünglicher Systemzustand wird wieder hergestellt"

#: ../DistUpgrade/DistUpgradeController.py:1151
#: ../DistUpgrade/DistUpgradeController.py:1166
#: ../DistUpgrade/DistUpgradeController.py:1188
msgid "Could not install the upgrades"
msgstr "Die Aktualisierungen konnten nicht installiert werden"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1156
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Die Systemaktualisierung wurde abgebrochen. Ihr System könnte sich in einem "
"nicht verwendbaren Zustand befinden. Eine Wiederherstellung wird gestartet "
"(»dpkg --configure -a«)."

#: ../DistUpgrade/DistUpgradeController.py:1161
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Bitte melden Sie diesen Fehler in einem Browser unter "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"und hängen Sie die Dateien in /var/log/dist-upgrade/ an den Problembericht "
"an.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1198
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Die Systemaktualisierung wurde abgebrochen. Bitte prüfen Sie Ihre "
"Internetverbindung oder Ihr Installationsmedium und versuchen Sie es erneut. "

#: ../DistUpgrade/DistUpgradeController.py:1278
msgid "Remove obsolete packages?"
msgstr "Veraltete Pakete entfernen?"

#: ../DistUpgrade/DistUpgradeController.py:1279
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Beibehalten"

#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "_Remove"
msgstr "_Entfernen"

#: ../DistUpgrade/DistUpgradeController.py:1291
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Während des Aufräumens trat ein Problem auf. Bitte lesen Sie die unten "
"stehende Meldung für weitere Informationen. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1367
msgid "Required depends is not installed"
msgstr ""
"Die aufgrund von Abhängigkeiten notwendigen Pakete sind nicht installiert."

#: ../DistUpgrade/DistUpgradeController.py:1368
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr ""
"Das aufgrund von Abhängigkeiten erforderliche Paket »%s« ist nicht "
"installiert. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1701
msgid "Checking package manager"
msgstr "Paketverwaltung wird überprüft"

#: ../DistUpgrade/DistUpgradeController.py:1641
msgid "Preparing the upgrade failed"
msgstr "Vorbereitung der Systemaktualisierung gescheitert"

#: ../DistUpgrade/DistUpgradeController.py:1642
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Die Vorbereitung des Systems zur Aktualisierung ist fehlgeschlagen, die "
"Fehlerberichterstattung wurde somit gestartet."

#: ../DistUpgrade/DistUpgradeController.py:1656
msgid "Getting upgrade prerequisites failed"
msgstr "Beziehen der Erfordernisse für die Systemaktualisierung gescheitert"

#: ../DistUpgrade/DistUpgradeController.py:1657
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Das System konnte nicht die Erfordernisse für die Systemaktualisierung "
"beziehen. Die Systemaktualisierung wird nun abgebrochen und das "
"ursprüngliche System wiederhergestellt.\n"
"\n"
"Außerdem wird der Fehlermeldevorgang gestartet."

#: ../DistUpgrade/DistUpgradeController.py:1685
msgid "Updating repository information"
msgstr "Informationen zu Paketquellen werden aktualisiert"

#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Failed to add the cdrom"
msgstr "CD-ROM konnte nicht hinzugefügt werden"

#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Sorry, adding the cdrom was not successful."
msgstr "Entschuldigung, das Hinzufügen der CD-ROM ist fehlgeschlagen."

#: ../DistUpgrade/DistUpgradeController.py:1724
msgid "Invalid package information"
msgstr "Ungültige Paketinformationen"

#: ../DistUpgrade/DistUpgradeController.py:1725
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Das benötigte Paket »%s« wurde nach der Aktualisierung Ihrer "
"Paketinformationen nicht gefunden. Dies kann daran liegen, dass Sie keine "
"offiziellen Spiegel-Server in Ihren Paketquellen eingetragen haben oder ein "
"Spiegel-Server überlastet ist.\n"
"Ihre derzeit eingerichteten Paketquellen finden Sie in "
"/etc/apt/sources.list. Im Falle eines überlasteten Spiegel-Servers können "
"Sie es später erneut versuchen."

#: ../DistUpgrade/DistUpgradeController.py:1749
#: ../DistUpgrade/DistUpgradeController.py:1805
msgid "Fetching"
msgstr "Herunterladen"

#: ../DistUpgrade/DistUpgradeController.py:1755
#: ../DistUpgrade/DistUpgradeController.py:1809
msgid "Upgrading"
msgstr "Systemaktualisierung wird durchgeführt"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1760
#: ../DistUpgrade/DistUpgradeController.py:1811
#: ../DistUpgrade/DistUpgradeController.py:1818
#: ../DistUpgrade/DistUpgradeController.py:1829
msgid "Upgrade complete"
msgstr "Systemaktualisierung wurde abgeschlossen"

#: ../DistUpgrade/DistUpgradeController.py:1761
#: ../DistUpgrade/DistUpgradeController.py:1812
#: ../DistUpgrade/DistUpgradeController.py:1819
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Die Systemaktualisierung wurde vollständig abgeschlossen, jedoch traten "
"während der Systemaktualisierung Fehler auf."

#: ../DistUpgrade/DistUpgradeController.py:1768
msgid "Searching for obsolete software"
msgstr "Es wird nach veralteter Software gesucht"

#: ../DistUpgrade/DistUpgradeController.py:1781
msgid "System upgrade is complete."
msgstr "Die Systemaktualisierung ist abgeschlossen."

#: ../DistUpgrade/DistUpgradeController.py:1830
msgid "The partial upgrade was completed."
msgstr "Teilweise Systemaktualisierung abgeschlossen."

#: ../DistUpgrade/DistUpgradeFetcher.py:123
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:119
msgid "Could not find the release notes"
msgstr "Versionshinweise konnten nicht gefunden werden"

#: ../DistUpgrade/DistUpgradeFetcher.py:124
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:120
msgid "The server may be overloaded. "
msgstr "Der Server ist möglicherweise überlastet. "

#: ../DistUpgrade/DistUpgradeFetcher.py:136
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:124
msgid "Could not download the release notes"
msgstr "Versionshinweise konnten nicht heruntergeladen werden"

#: ../DistUpgrade/DistUpgradeFetcher.py:137
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:125
msgid "Please check your internet connection."
msgstr "Bitte überprüfen Sie Ihre Internetverbindung."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "»%(file)s« wird gegenüber »%(signature)s« legitimiert "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#, python-format
msgid "extracting '%s'"
msgstr "»%s« wird entpackt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:157
#: ../DistUpgrade/DistUpgradeFetcherCore.py:158
msgid "Could not run the upgrade tool"
msgstr "Die Aktualisierungsanwendung konnte nicht gestartet werden"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:159
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Hierbei handelt es sich sehr wahrscheinlich im einen Fehler im "
"Aktualisierungswerkzeug. Bitter berichten Sie den Fehler mit dem Befehl "
"»ubuntu-bug ubuntu-release-upgrader-core«."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:236
msgid "Upgrade tool signature"
msgstr "Signatur der Aktualisierungsanwendung"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Upgrade tool"
msgstr "Aktualisierungsanwendung"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to fetch"
msgstr "Herunterladen gescheitert"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Das Herunterladen der Systemaktualisierung ist gescheitert. Möglicherweise "
"gibt es ein Netzwerkproblem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:282
msgid "Authentication failed"
msgstr "Legitimierung gescheitert"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Die Systemaktualisierung konnten nicht auf ihre Echtheit überprüft werden. "
"Möglicherweise gibt es Probleme mit dem Netzwerk oder mit dem Server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:288
msgid "Failed to extract"
msgstr "Entpacken gescheitert"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:289
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Die Systemaktualisierung konnte nicht entpackt werden. Möglicherweise gibt "
"es ein Problem mit dem Netzwerk oder mit dem Server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:294
msgid "Verification failed"
msgstr "Überprüfung gescheitert"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:295
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Die Überprüfung der Systemaktualisierung ist gescheitert. Möglicherweise "
"gibt es ein Problem mit dem Netzwerk oder mit dem Server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:309
#: ../DistUpgrade/DistUpgradeFetcherCore.py:315
msgid "Can not run the upgrade"
msgstr "Systemaktualisierung kann nicht durchgeführt werden"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:310
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dies tritt normalerweise bei einem System auf, in dem /tmp als noexec "
"eingehängt ist. Bitte hängen Sie /tmp ohne noexec ein und führen Sie die "
"Aktualisierung erneut aus."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:316
#, python-format
msgid "The error message is '%s'."
msgstr "Die Fehlermeldung lautet »%s«."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:70
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:101
msgid "Upgrade"
msgstr "Systemaktualisierung"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:105
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Veröffentlichungshinweise"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:147
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:160
msgid "Downloading additional package files..."
msgstr "Zusätzliche Pakete werden heruntergeladen …"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:162
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Datei %s von %s mit %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:166
#, python-format
msgid "File %s of %s"
msgstr "Datei %s von %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:173
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:115
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Bitte legen Sie »%s« in das Laufwerk »%s« ein"

#. change = QMessageBox.question(None, _("Media Change"), msg,
#. QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:177
#: ../DistUpgrade/DistUpgradeViewKDE.py:197
msgid "Media Change"
msgstr "Medienwechsel"

#: ../DistUpgrade/DistUpgradeQuirks.py:208
msgid "evms in use"
msgstr "evms ist in Verwendung"

#: ../DistUpgrade/DistUpgradeQuirks.py:209
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Ihr System verwendet die »evms«-Datenträgerverwaltung in /proc/mounts. Die "
"Anwendung »evms« wird nicht mehr unterstützt. Bitte schalten Sie »emvs« ab "
"und starten Sie die Systemaktualisierung danach erneut."

#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Your graphics hardware may not be fully supported in Ubuntu 13.04."
msgstr ""
"Ihre Grafik-Hardware wird möglicherweise von Ubuntu 13.04 nicht vollständig "
"unterstützt."

#: ../DistUpgrade/DistUpgradeQuirks.py:248
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""
"Das Ausführen der »Unity«-Arbeitsumgebung wird nicht vollständig von Ihrer "
"Grafik-Hardware unterstützt. Dies könnte nach der Aktualisierung zu einer "
"sehr langsamen Umgebung führen. Wir raten Ihnen, statdessen die LTS-Version "
"zu benutzen. Für weitere Informationen besuchen Sie: "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Möchten Sie "
"die Aktualisierung trotzdem fortsetzen?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Ihre Grafik-Hardware wird möglicherweise nicht vollständig von Ubuntu 12.04 "
"LTS unterstützt."

#: ../DistUpgrade/DistUpgradeQuirks.py:274
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Ihre Intel-Grafik-Hardware wird in Ubuntu 12.04 LTS nur begrenzt unterstützt "
"und Sie werden nach der Systemaktualisierung möglicherweise auf Probleme "
"stoßen. Weitere Informationen dazu finden Sie unter "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Möchten Sie die "
"Systemaktualisierung fortsetzen?"

#: ../DistUpgrade/DistUpgradeQuirks.py:294
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Eine Systemaktualisierung kann zu eingeschränkten visuellen Effekte führen "
"und die Geschwindigkeit in Spielen und anderen grafikintensiven Anwendungen "
"herabsetzen."

#: ../DistUpgrade/DistUpgradeQuirks.py:564
#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Der Recher verwendet derzeit den Gafiktreiber »nvidia« von NVIDIA. Es ist "
"keine Version dieses Treiber verfügbar, die mit Ihrer Grafikkarte in Ubuntu "
"10.04 LTS arbeiten kann.\n"
"\n"
"Möchten Sie fortfahren?"

#: ../DistUpgrade/DistUpgradeQuirks.py:298
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Der Rechner verwendet derzeit den Grafiktreiber »fglrx« von AMD. Es ist "
"keine Version dieses Treiber verfügbar, die mit Ihrer Hardware in Ubuntu "
"10.04 LTS arbeiten kann.\n"
"\n"
"Möchten Sie fortfahren?"

#: ../DistUpgrade/DistUpgradeQuirks.py:333
msgid "No i686 CPU"
msgstr "Kein i686-Prozessor vorhanden"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ihr Rechner verwendet einen i586-Prozessor oder einen Prozessor, der keine "
"CMOV-Instruktionen unterstützt. Alle Pakete wurden jedoch so optimiert, dass "
"sie mindestens einen i686-Prozessor erfordern. Deshalb ist es mit dieser "
"Hardware nicht möglich, Ihren Rechner auf eine neue Ubuntu-Version zu "
"aktualisieren."

#: ../DistUpgrade/DistUpgradeQuirks.py:370
msgid "No ARMv6 CPU"
msgstr "Kein ARMv6-Prozessor"

#: ../DistUpgrade/DistUpgradeQuirks.py:371
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ihr System verwendet eine ARM-CPU mit einer älteren Architektur als ARMv6. "
"Alle Softwarepakete in Karmic wurden mit Optimierungen erstellt, die ARMv6 "
"als minimale Architektur erfordern. Es ist nicht möglich, mit dieser "
"Hardware Ihr System auf eine neue Ubuntu-Version zu aktualisieren."

#: ../DistUpgrade/DistUpgradeQuirks.py:391
msgid "No init available"
msgstr "Kein init verfügbar"

#: ../DistUpgrade/DistUpgradeQuirks.py:392
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Ihr System scheint eine virtuelle Umgebung ohne init-daemon zu sein, z.B. "
"Linux-VServer. Ubuntu 10.04 LTS läuft in dieser Art von Umgebung nicht, "
"sondern benötigt zuerst eine Aktualisierung der Konfiguration Ihrer "
"virtuellen Maschine.\n"
"\n"
"Sind Sie sicher, dass Sie fortfahren möchten?"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Sandbox upgrade using aufs"
msgstr "Sandbox-Systemaktualisierung mit »aufs«"

#: ../DistUpgrade/DistUpgradeMain.py:66
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Den angegebenen Pfad benutzen, um nach einer CD-ROM mit aktualisierbaren "
"Paketen zu suchen."

#: ../DistUpgrade/DistUpgradeMain.py:72
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Grafische Benutzeroberfläche verwenden. Momentan verfügbar: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "*DEPRECATED* this option will be ignored"
msgstr "*VERALTET* diese Option wird ignoriert"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Nur teilweise Systemaktualisierung durchführen (ohne sources.list erneut zu "
"schreiben)"

#: ../DistUpgrade/DistUpgradeMain.py:81
msgid "Disable GNU screen support"
msgstr "GNU-Bildschirmunterstützung deaktivieren"

#: ../DistUpgrade/DistUpgradeMain.py:83
msgid "Set datadir"
msgstr "Datenverzeichnis festlegen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:136
#: ../DistUpgrade/DistUpgradeViewKDE.py:210
msgid "Fetching is complete"
msgstr "Dateien wurden vollständig heruntergeladen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk3.py:151
#: ../DistUpgrade/DistUpgradeViewKDE.py:226
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Datei %li von %li wird mit %sB/s heruntergeladen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:151
#: ../DistUpgrade/DistUpgradeViewGtk.py:298
#: ../DistUpgrade/DistUpgradeViewGtk3.py:153
#: ../DistUpgrade/DistUpgradeViewGtk3.py:310
#: ../DistUpgrade/DistUpgradeViewKDE.py:227
#: ../DistUpgrade/DistUpgradeViewKDE.py:375
#, python-format
msgid "About %s remaining"
msgstr "Es verbleiben ungefähr %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:154
#: ../DistUpgrade/DistUpgradeViewGtk3.py:156
#: ../DistUpgrade/DistUpgradeViewKDE.py:229
#, python-format
msgid "Fetching file %li of %li"
msgstr "Datei %li von %li wird heruntergeladen"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:185
#: ../DistUpgrade/DistUpgradeViewGtk3.py:187
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "Applying changes"
msgstr "Änderungen werden übernommen"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:210
#: ../DistUpgrade/DistUpgradeViewGtk3.py:213
#: ../DistUpgrade/DistUpgradeViewKDE.py:279
msgid "dependency problems - leaving unconfigured"
msgstr "Abhängigkeitsprobleme - beende unkonfiguriert"

#: ../DistUpgrade/DistUpgradeViewGtk.py:215
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:281
#, python-format
msgid "Could not install '%s'"
msgstr "»%s« konnte nicht installiert werden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:216
#: ../DistUpgrade/DistUpgradeViewGtk3.py:219
#: ../DistUpgrade/DistUpgradeViewKDE.py:282
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Die Systemaktualisierung wird fortgesetzt, aber das Paket »%s« könnte sich "
"in einem nicht funktionsfähigen Zustand befinden. Bitte ziehen Sie in "
"Betracht, diesen Fehler zu melden."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:233
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Soll die manuell angepasste Konfigurationsdatei\n"
"%s ersetzt werden?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:234
#: ../DistUpgrade/DistUpgradeViewGtk3.py:237
#: ../DistUpgrade/DistUpgradeViewKDE.py:304
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Alle Änderungen an dieser Konfigurationsdatei gehen verloren, wenn Sie diese "
"durch eine neuere Version ersetzen."

#: ../DistUpgrade/DistUpgradeViewGtk.py:253
#: ../DistUpgrade/DistUpgradeViewGtk3.py:257
#: ../DistUpgrade/DistUpgradeViewKDE.py:327
msgid "The 'diff' command was not found"
msgstr "Das Programm »diff« konnte nicht gefunden werden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:466
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
#: ../DistUpgrade/DistUpgradeViewText.py:100
msgid "A fatal error occurred"
msgstr "Ein schwerwiegender Fehler ist aufgetreten"

#: ../DistUpgrade/DistUpgradeViewGtk.py:467
#: ../DistUpgrade/DistUpgradeViewGtk3.py:479
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Bitte erstellen Sie einen Fehlerbericht (wenn sie dies noch nicht getan "
"haben) und fügen Sie die Dateien /var/log/dist-upgrade/main.log und "
"/var/log/dist-upgrade/apt.log hinzu. Die Systemaktualisierung wurde "
"abgebrochen. Ihre ursprüngliche sources.list wurde unter "
"/etc/apt/sources.list.distUpgrade gespeichert."

#: ../DistUpgrade/DistUpgradeViewGtk.py:484
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid "Ctrl-c pressed"
msgstr "Strg-C wurde gedrückt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:485
#: ../DistUpgrade/DistUpgradeViewGtk3.py:497
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dies führt zum Abbruch der Operation und eventuell zu einem instabilen "
"System. Sind Sie sicher, dass Sie das tun möchten?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:633
#: ../DistUpgrade/DistUpgradeViewGtk3.py:630
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Schließen Sie bitte alle offenen Anwendungen und Dokumente, um Datenverluste "
"zu vermeiden."

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Nicht mehr von Canonical unterstützt (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Mit älterer Version ersetzen (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "Remove (%s)"
msgstr "Entfernen (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "No longer needed (%s)"
msgstr "Nicht mehr benötigt (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:651
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Install (%s)"
msgstr "Installieren (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:652
#: ../DistUpgrade/DistUpgradeViewGtk3.py:649
#, python-format
msgid "Upgrade (%s)"
msgstr "Aktualisieren (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:339
msgid "Show Difference >>>"
msgstr "Unterschiede anzeigen >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:342
msgid "<<< Hide Difference"
msgstr "<<< Unterschiede ausblenden"

#: ../DistUpgrade/DistUpgradeViewKDE.py:558
msgid "Error"
msgstr "Fehler"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
msgid "&Cancel"
msgstr "Ab&brechen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:817
msgid "&Close"
msgstr "&Schließen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:622
msgid "Show Terminal >>>"
msgstr "Terminal anzeigen >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:625
msgid "<<< Hide Terminal"
msgstr "<<< Terminal ausblenden"

#: ../DistUpgrade/DistUpgradeViewKDE.py:705
msgid "Information"
msgstr "Information"

#: ../DistUpgrade/DistUpgradeViewKDE.py:755
#: ../DistUpgrade/DistUpgradeViewKDE.py:800
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Details"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#, python-format
msgid "No longer supported %s"
msgstr "Nicht mehr unterstützt (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Remove %s"
msgstr "Entferne %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#: ../DistUpgrade/DistUpgradeViewText.py:194
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Entferne (wurde automatisch installiert): %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:787
#, python-format
msgid "Install %s"
msgstr "Installiere %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:789
#, python-format
msgid "Upgrade %s"
msgstr "Aktualisiere %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
#: ../DistUpgrade/DistUpgradeViewText.py:242
msgid "Restart required"
msgstr "Neustart erforderlich"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Das System zum Abschluss der Systemaktualisierung neu "
"starten</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:816
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "Jetzt _neu starten"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Die laufende Systemaktualisierung abbrechen?</big></b>\n"
"\n"
"Das System kann sich in einem unbenutzbaren Zustand befinden, wenn Sie die "
"Aktualisierung abbrechen. Es wird dringend empfohlen, die "
"Systemaktualisierung fortzusetzen."

#: ../DistUpgrade/DistUpgradeViewKDE.py:838
msgid "Cancel Upgrade?"
msgstr "Systemaktualisierung abbrechen?"

#: ../DistUpgrade/DistUpgradeView.py:73
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li Tag"
msgstr[1] "%li Tage"

#: ../DistUpgrade/DistUpgradeView.py:75
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li Stunde"
msgstr[1] "%li Stunden"

#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li Minute"
msgstr[1] "%li Minuten"

#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li Sekunde"
msgstr[1] "%li Sekunden"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:94
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:112
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:163
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Dieser Download wird mit einer 1-MBit-DSL-Verbindung etwa %s dauern oder "
"ungefähr %s mit einem 56K-Modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:167
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Das Herunterladen wird bei Ihrer Netzwerkverbindung etwa %s dauern. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:271
#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Systemaktualisierung wird vorbereitet"

#: ../DistUpgrade/DistUpgradeView.py:272
msgid "Getting new software channels"
msgstr "Neue Paketquellen werden abgefragt"

#: ../DistUpgrade/DistUpgradeView.py:273
#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Aktualisierungen werden heruntergeladen"

#: ../DistUpgrade/DistUpgradeView.py:274
#: ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Aktualisierungen werden installiert"

#: ../DistUpgrade/DistUpgradeView.py:275
#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Aufräumen"

#: ../DistUpgrade/DistUpgradeView.py:360
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d Paket wird nicht mehr von Canonical unterstützt. Sie können "
"allerdings immer noch Hilfe von der Gemeinschaft erhalten."
msgstr[1] ""
"%(amount)d Pakete werden nicht mehr von Canonical unterstützt. Sie können "
"allerdings immer noch Hilfe von der Gemeinschaft erhalten."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:369
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d Paket wird entfernt."
msgstr[1] "%d Pakete werden entfernt."

#: ../DistUpgrade/DistUpgradeView.py:374
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d neues Paket wird installiert."
msgstr[1] "%d neue Pakete werden installiert."

#: ../DistUpgrade/DistUpgradeView.py:380
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d Paket wird aktualisiert."
msgstr[1] "%d Pakete werden aktualisiert."

#: ../DistUpgrade/DistUpgradeView.py:388
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Insgesamt müssen %s heruntergeladen werden. "

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Die Installieren der Systemaktualisierung kann mehrere Stunden dauern. "
"Sobald das Herunterladen abgeschlossen wurde, kann der Vorgang nicht "
"abgebrochen werden."

#: ../DistUpgrade/DistUpgradeView.py:397
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Das Herunterladen und Installieren der Systemaktualisierung kann mehrere "
"Stunden dauern. Sobald das Herunterladen abgeschlossen ist, kann der Vorgang "
"nicht mehr abgebrochen werden."

#: ../DistUpgrade/DistUpgradeView.py:402
msgid "Removing the packages can take several hours. "
msgstr "Das Entfernen der Pakete kann mehrere Stunden dauern. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:407
msgid "The software on this computer is up to date."
msgstr "Die Anwendungen auf diesem Rechner sind aktuell."

#: ../DistUpgrade/DistUpgradeView.py:408
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Es sind keine Aktualisierungen für das System verfügbar. Der Vorgang wird "
"nun beendet."

#: ../DistUpgrade/DistUpgradeView.py:421
msgid "Reboot required"
msgstr "Neustart erforderlich"

#: ../DistUpgrade/DistUpgradeView.py:422
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Die Systemaktualisierung ist abgeschlossen und ein Neustart ist "
"erforderlich. Möchten Sie den Computer jetzt neu starten?"

#: ../DistUpgrade/DistUpgradeViewText.py:101
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Bitte erstellen Sie einen Fehlerbericht und fügen Sie die Dateien "
"/var/log/dist-upgrade/main.log und /var/log/dist-upgrade/apt.log hinzu. Die "
"Systemaktualisierung wurde abgebrochen.\n"
"Ihre ursprüngliche sources.list wurde in /etc/apt/sources.list.distUpgrade "
"gesichert."

#: ../DistUpgrade/DistUpgradeViewText.py:125
msgid "Aborting"
msgstr "Wird abgebrochen"

#: ../DistUpgrade/DistUpgradeViewText.py:130
msgid "Demoted:\n"
msgstr "Zurückgesetzt:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "To continue please press [ENTER]"
msgstr "Um fortzufahren, drücken Sie [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
#: ../DistUpgrade/DistUpgradeViewText.py:215
msgid "Continue [yN] "
msgstr "Fortsetzen [j/N] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "Details [d]"
msgstr "Details [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:174
#: ../DistUpgrade/DistUpgradeViewText.py:218
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:177
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:180
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "No longer supported: %s\n"
msgstr "Nicht mehr unterstützt: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Remove: %s\n"
msgstr "»%s« wird entfernt\n"

#: ../DistUpgrade/DistUpgradeViewText.py:200
#, python-format
msgid "Install: %s\n"
msgstr "»%s« wird installiert\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
#, python-format
msgid "Upgrade: %s\n"
msgstr "»%s« wird aktualisiert\n"

#: ../DistUpgrade/DistUpgradeViewText.py:222
msgid "Continue [Yn] "
msgstr "Fortsetzen [J/n] "

#: ../DistUpgrade/DistUpgradeViewText.py:243
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Um die Systemaktualisierung abzuschließen, ist ein Neustart erforderlich.\n"
"Wenn Sie »j« wählen, wird das System neu gestartet."

#: ../DistUpgrade/GtkProgress.py:73
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"Datei %(current)li von %(total)li wird mit %(speed)s/s heruntergeladen"

#: ../DistUpgrade/GtkProgress.py:79
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Datei %(current)li von %(total)li wird heruntergeladen"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Fortschritt der einzelnen Dateien anzeigen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "Systemaktualisierung abbre_chen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "System_aktualisierung fortsetzen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Soll die laufende Systemaktualisierung abgebrochen "
"werden?</big></b>\n"
"\n"
"Das System könnte in einem unbenutzbaren Zustand verbleiben. Es wird "
"dringend geraten, die Systemaktualisierung fortzusetzen."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "Systemaktualisierung _beginnen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Ersetzen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Unterschiede zwischen den Dateien"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Fehlerbericht erstellen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Fortsetzen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Mit der Systemaktualisierung beginnen?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Das System muss neu gestartet werden, um die Aktualisierung "
"abzuschließen.</big></b>\n"
"\n"
"Bitte sichern Sie alle offenen Arbeiten, bevor Sie fortfahren."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Systemaktualisierung"

#: ../data/gtkbuilder/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 13.04</big></b>"
msgstr "<b><big>Aktualisierung von Ubuntu auf Version 13.04</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Neue Paketquellen werden eingerichtet"

#: ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Rechner wird neu gestartet"

#: ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Befehlsfenster"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_System aktualisieren"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Eine neue Ubuntu-Version ist verfügbar. Möchten Sie Ihr System "
"aktualisieren?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "System nicht aktualisieren"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Später erneut fragen"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "System jetzt aktualisieren"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr ""
"Sie haben die Systemaktualisierung auf eine neue Ubuntu-Veresion abgelehnt"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Sie können die Aktualisierung auch später durchführen, indem Sie die "
"Anwendung »Aktualisierungen« öffnen und dort auf »Aktualisieren« klicken."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Eine Systemaktualisierung durchführen"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Um Ubuntu zu aktualisieren, müssen Sie sich authentifizieren."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Eine teilweise Systemaktualisierung durchführen"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""
"Um eine teilweise Systemaktualisierung durchzuführen, müssen Sie sich "
"authentifizieren."

#: ../do-partial-upgrade:67 ../do-release-upgrade:69
msgid "Show version and exit"
msgstr "Version anzeigen und Programm schließen"

#: ../do-partial-upgrade:70 ../do-release-upgrade:76
msgid "Directory that contains the data files"
msgstr "Verzeichnis, das die Datendateien enthält"

#: ../do-partial-upgrade:73 ../do-release-upgrade:89
msgid "Run the specified frontend"
msgstr "Angegebene Oberfläche verwenden"

#: ../do-partial-upgrade:90
msgid "Running partial upgrade"
msgstr "Teilweise Systemaktualisierung wird durchgeführt"

#: ../do-release-upgrade:31
msgid "Downloading the release upgrade tool"
msgstr "Das Freigabeaktualisierungswerkzeug wird heruntergeladen"

#: ../do-release-upgrade:72 ../check-new-release-gtk:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Möglichkeit einer Aktualisierung auf die aktuelle Entwicklungsversion prüfen"

#: ../do-release-upgrade:79 ../check-new-release-gtk:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Versuchen Sie, mit der Aktualisierungsverwaltung von $distro-proposed auf "
"die neueste Version zu aktualisieren."

#: ../do-release-upgrade:83
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Verwendung eines besonderen Systemaktualisierungsmodus\n"
"Aktuell werden die Modi »desktop« für die allgemeine Systemaktualisierung "
"von Desktop-Systemen und »server« für Server-Systeme unterstützt."

#: ../do-release-upgrade:91
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Testen der Systemaktualisierung mit einem Sandbox-aufs-Overlay"

#: ../do-release-upgrade:94
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Nur auf neue Systemfreigabe prüfen und das Ergebnis im Exit-Code angebe"

#: ../do-release-upgrade:108
msgid "Checking for a new Ubuntu release"
msgstr "Neue Veröffentlichungen von Ubuntu werden gesucht"

#: ../do-release-upgrade:121
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ihre Ubuntu-Version wird nicht länger unterstützt."

#: ../do-release-upgrade:122
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Bitte rufen Sie diese Webseite auf, um Informationen zur "
"Systemaktualisierung zu erhalten:\n"
"%(url)s\n"

#: ../do-release-upgrade:128
msgid "No new release found"
msgstr "Keine neue Freigabe gefunden"

#: ../do-release-upgrade:133
msgid "Release upgrade not possible right now"
msgstr "Eine Systemaktualisierung ist momentan nicht möglich"

#: ../do-release-upgrade:134
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Die Systemaktualisierung kann im Moment nicht durchgeführt werden, bitte "
"versuchen Sie es später erneut. Server-Meldung: »%s«"

#: ../do-release-upgrade:140
#, c-format
msgid "New release '%s' available."
msgstr "Neue Freigabe »%s« verfügbar."

#: ../do-release-upgrade:141
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr ""
"Führen Sie den Befehl »do-release-upgrade« aus, um auf die neue Ubuntu-"
"Freigabe zu aktualisieren."

#: ../check-new-release-gtk:113
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s-Systemaktualisierung verfügbar"

#: ../check-new-release-gtk:144
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Sie haben die Systemaktualisierung auf Ubuntu %s abgelehnt"

#: ../check-new-release-gtk:186
msgid "Add debug output"
msgstr "Fehlerdiagnoseausgabe hinzufügen"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr ""
#~ "Für die teilweise Systemaktualisierung ist eine Legitimation erforderlich"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Für die Systemaktualiserung ist eine Legitimation erforderlich"
